source/geometry/Quaternion.ooc

```diff
@@ -218,35 +215,29 @@ Quaternion: cover {
 	createFromEulerAngles: static func (rotationX, rotationY, rotationZ: Float) -> This {
 		This createRotationZ(rotationZ) * This createRotationY(rotationY) * This createRotationX(rotationX)
 	}
-	createRotation: static func (angle: Float, direction: FloatPoint3D) -> This {
-		halfAngle := angle / 2.0f
-		point3DNorm := direction norm
-		if (point3DNorm != 0.0f)
-			direction /= point3DNorm
-		This new(0.0f, halfAngle * direction) exponential
-	}
-	createRotationX: static func (angle: Float) -> This {
-		This createRotation(angle, FloatPoint3D new(1.0f, 0.0f, 0.0f))
-	}
-	createRotationY: static func (angle: Float) -> This {
-		This createRotation(angle, FloatPoint3D new(0.0f, 1.0f, 0.0f))
-	}
-	createRotationZ: static func (angle: Float) -> This {
-		This createRotation(angle, FloatPoint3D new(0.0f, 0.0f, 1.0f))
-	}
-	relativeFromVelocity: static func (angularVelocity: FloatPoint3D) -> This {
+	createFromAxisAngle: static func (rotation: FloatVector3D) {
 		result := This identity
-		angle := sqrt(angularVelocity x * angularVelocity x + angularVelocity y * angularVelocity y + angularVelocity z * angularVelocity z)
+		angle := rotation norm
 		if (angle > 1.0e-8f) {
 			result = This new(
 				cos(angle / 2.0f),
-				angularVelocity x * sin(angle / 2.0f) / angle,
-				angularVelocity y * sin(angle / 2.0f) / angle,
-				angularVelocity z * sin(angle / 2.0f) / angle
+				rotation x * sin(angle / 2.0f) / angle,
+				rotation y * sin(angle / 2.0f) / angle,
+				rotation z * sin(angle / 2.0f) / angle
 				)
 		}
 		result
 	}
+	createRotation: static func (angle: Float, direction: FloatPoint3D) -> This {
```

Comment:
It seems this function does the same as createFromAxisAngle. Can we confirm and remove one of them?